src/Bindings/DeprecatedBindings.cpp

```diff
@@ -141,12 +141,12 @@ static int tolua_get_AllToLua_g_BlockPistonBreakable(lua_State* tolua_S)
 		}
 	}
 	#endif
-	BlockType = (int)tolua_tonumber(tolua_S, 2, 0);
+	BlockType = static_cast<int>(tolua_tonumber(tolua_S, 2, 0));
 	if ((BlockType < 0) || (BlockType > E_BLOCK_MAX_TYPE_ID))
 	{
 		tolua_error(tolua_S, "array indexing out of range.", nullptr);
 	}
-	tolua_pushboolean(tolua_S, cBlockInfo::IsPistonBreakable((BLOCKTYPE)BlockType));
+	tolua_pushboolean(tolua_S, cBlockInfo::IsPistonBreakable(static_cast<BLOCKTYPE>(BlockType)));
```

Comment:
Why not change the type of `BlockType` to `BLOCKTYPE` to avoid the cast altogether?